src/commands/txCommands/util/pushTxAllTranslations.js

```
@@ -0,0 +1,12 @@
+const chalk = require('chalk');
+const Rx = require('rxjs');
+const txlib = require('./index');
+
+const pushTxAllTranslations = () => {
+  txlib.checkEnvVars();
+  console.log(chalk.blue('pushing content to all_translations in mup-web'));
+  Rx.Observable
+    .concat(txlib.updateAllTranslationsResource$)
+    .subscribe(null, null, () => console.log('done'));
```

this will swallow errors - better to `console.error` them